src/entity/projectile/WindCharge.php

```diff
@@ -41,8 +41,6 @@
 use function round;
 
 class WindCharge extends Throwable{
-	protected float $radius = 2.5;
```

Comment:
I told you about making them constants, no remvonving them

This is a constant btw
https://github.com/pmmp/PocketMine-MP/blob/585aee9386a787c95e73dd0a05ffca8329606b68/src/entity/projectile/Arrow.php#L52